src/e2e-app/mdc-card/mdc-card-e2e.ts

```diff
@@ -10,8 +10,8 @@ import {Component} from '@angular/core';
 
 @Component({
   selector: 'mdc-card-e2e',
-  templateUrl: 'mdc-card-e2e.html',
+  template: `<example-list-viewer [ids]="examples"></example-list-viewer>`,
 })
 export class MdcCardE2e {
-  // TODO: copy implementation from existing mat-card e2e page.
+  examples = ['mdc-card-fancy'];
```

Comment:
Just as a note: I think it might be usually more convenient just use the example component directly. There should be no need to go through the lazy loading.

Reply:
+1 that would be much clearer

Reply:
Weird, CI doesn't like it but passes locally in VE